doc/docs/download_cfd.md

```diff
@@ -3,28 +3,37 @@
 Download nRF Connect for Desktop's executable from Nordic Semiconductor's [website](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Connect-for-desktop).
 
 All nRF Connect for Desktop applications require the launcher v4.1.0 or later.
+See also the list of [Supported operating systems](./os_support.md).
 
 After installing and starting the launcher, you will see the list of available apps.
 See the [Launcher overview](overview_cfd.md) and [Installing nRF Connect for Desktop apps](installing_apps.md) pages for more information.
 
 ## Additional requirements
 
-Running nRF Connect for Desktop on macOS and Linux has additional requirements.
+Running nRF Connect for Desktop has the following additional requirements.
 
-### J-Link driver on macOS and Linux
+### SEGGER J-Link driver
 
-You must install SEGGER J-Link driver separately on macOS and Linux. To do so, download and
-install it from [SEGGER](https://www.segger.com/downloads/jlink) under the
-section _J-Link Software and Documentation Pack_.
+Required on all platforms.
```

Comment:
@KievDevel , can you confirm this is valid for all platforms, not only for macOS and Linux? I'm mirroring https://docs.nordicsemi.com/bundle/nrfutil/page/guides/installing.html#prerequisites here.

Reply:
Main difference is that on Windows SEGGER J-Link is shipped with the app (bundled), but on macOS & Linux platforms user must install it separately.
But yes, it is required on all three platforms. Also there might be a problem downloading required version of the JLink as now they hide all minor releases, except the latest one.